SPECS/kernel/kernel.spec

```diff
@@ -155,7 +156,7 @@ manipulation of eBPF programs and maps.
 
 %prep
 %setup -q -n CBL-Mariner-Linux-Kernel-rolling-lts-mariner-%{mariner_version}-%{version}
-
+%patch1 -p1
```

Comment:
wait, does this mean we weren't actually applying Patch0 before? This should probably be changed over to %autosetup so we don't miss future patches, right?

Reply:
I intentionally turned off the nvme patch for 3.0 but have a task to see whether it should be kept. I agree this is confusing though as is. Let me yank out the patch